torch/_prims/__init__.py

```diff
@@ -1968,7 +1990,7 @@ def _reshape_meta(a: TensorLikeType, shape: ShapeType):
 
 
 def _reshape_aten(a: Tensor, shape: ShapeType) -> Tensor:
-    return a.reshape(shape).contiguous().clone()
+    return a.reshape(shape).contiguous()
```

Comment:
reshape is a bit of a special op. We don't want to constrain it to the custom_op API

Reply:
Do you mean to add back the `.clone()`? Or to use the old custom op API?

Reply:
Oh I see, I misunderstood. We should add the clone back